src/Utils/Storage.js

```diff
@@ -0,0 +1,21 @@
+export default class Storage {
+    constructor() {
+        this.storage = {};
+    }
+
+    setStorage(storage) {
+        this.storage = storage;
+    }
+
+    getItem(key) {
+        this.storage.getItem(key);
+    }
+
+    removeItem(key) {
+        this.storage.removeItem(key);
+    }
+
+    setItem(key, value) {
+        this.storage.setItem(key, value);
+    }
```

Comment:
trochę nie czaję czemu w klasach `StorageFactoryAbstract` i `Storage` mamy dokładnie takie metody, tj. `getItem`, `removeItem` i `setItem` 🤔

Reply:
Ja coś tutaj przeinżynierowałem grubo. Poprawiłem to wszystko i wrzuciłem ponownie, nie wiem jaki miałem zamysł wcześniej, ale chyba chciałem to zrobić za dobrze, a wyszło jak zawsze XD

Teraz powinno być ok

Reply:
relatable [']